site/_posts/2021-03-22-the-value-of-try-before-you-buy-excerpt.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog-post
+title: How to get the most out of a trial period
```

Comment:
How to get the most out of a free auth trial

Reply:
Will change to "How to get the most out of a free auth provider trial"